market/portal/meta.json

```diff
@@ -20,7 +20,7 @@
 			"archivedArtifacts": [
 				{
 				  "groupId": "ch.ivyteam.ivy.project.portal",
-				  "lastVersion": "10.0.19",
+				  "lastVersion": "10.1.0",
```

Comment:
keeping the legacy groupId is a good idea, this is LTS and we should not touch it.
but why do we have a minor version bump on an LTS train? @lttung-axonivy 
I think we should only do hotfix releases on LTS. Do we roll-out braking changes?

Reply:
Hi @ivy-rew 
AFAIK from team Ocpus, the problem is groupId of portal-app, portal-guide artifacts version 10 different from version 11/master. For LTS10, "groupId": "ch.ivyteam.ivy.project.portal". For LE, "groupId": "ch.ivyteam.ivy.project.portal". To solve the problem, we use "archivedArtifacts", "lastVersion".
- If Portal version is LESS than or EQUAL the "lastVersion", Market will collect portal-app, portal-guide artifacts by "groupId": "ch.ivyteam.ivy.project.portal". 
- If Portal version is GREATER than the "lastVersion", Market will collect portal-app, portal-guide artifacts by "groupId": "com.axonivy.portal".
Before this PR, "lastVersion": "10.0.19". Market Portal 10.0.19 works well, but with Portal 10.0.21, we cannot download portal-app, portal-guide because Market collects with incorrect groupId.
With **"lastVersion": "10.1.0"**, it means all Portal versions LTS10 10.0.x (less than "lastVersion": "10.1.0") will be collected with  "groupId": "ch.ivyteam.ivy.project.portal". LE version 11.x.x  (greater than "lastVersion": "10.1.0") will be collected with "groupId": "com.axonivy.portal" => this would work well for both LTS10 and LE

If you have other concerns related to this PR, please let us know.

Reply:
Thanks a lot for the clarification @lttung-axonivy . 
Glad to hear that portal on LTS 10 will keep to ship hotfix versions without changing major.minor "10.0.X".

For me then, the 'lastVersion' is "just" a new property to steer the version selection on the market ui.
I think we should think of a better name for it. To me it doesn't naturally tell that this is a max version selector.
So I'd ask for a introducing a the 'schema' for this new feature... maybe we can discuss on there how to name and describe this mechanism on it.

Reply:
Hi @ivy-rew, @nqhoan-axonivy, @ntqdinh-axonivy 
I think we need to release 10.0.21 ASAP, so IMO we should merge this PR soon. Then maybe you and team Octopus could discuss more about the solution. Currently, we cannot release Portal 10.0.21 due to this problem. Could you please give your ideas?

Reply:
Yes; I agree @lttung-axonivy , we can merge this and clean-up this feature in the upcoming sprint @ivy-sgi .